### ISSUE TITLE
Rescuezilla 2.4 - not booting correctly

### ISSUE THREAD

**Issue by CoodyCZ** [rescuezilla/rescuezilla#351]
After selecting language and trying first method - Start Rescuezilla - after selecting this, it is booting, but nothing is showing. Only black screen (version 2.4).

After selecting the second method - Graphical fallback mode - it's working properly.

Tryied from USB directly, and from USB created with Ventoy.

**Comment by CoodyCZ** [comment]
UPDATE: tryied in VMWare Workstation Pro ver. 16.2.4 - the same problem, if i select classic Start of rescuezilla, nothing shows. 
I have used clear downloaded ISO, loaded and booted from VMWare (UEFI, BIOS).

**Comment by engstk** [comment]
Can confirm this since the change to 22.04 base.

Tested on systems with intel and igpu, amd system and some amd gpus.
Some laptops also seem to be the same.
Only booting Fallback graphics or with nomodeset works atm.

**Comment by shasheene** [comment]
Oh man, sounds like I screwed up the video drivers on Rescuezilla v2.4 leading to blank screens on most graphics hardware  after I updated to Ubuntu 22.04 "Jammy".

I was hoping my new "weekly rolling releases" model would have helped catch issues like this before a stable release :disappointed: 

Expect an updated Rescuezilla v2.4.1 release hopefully this weekend.

**Comment by Free-Radical** [comment]
DITTO, black screen after boot

**Comment by hetzbh** [comment]
I just tested 2.4 with Lenovo SR630 server, both the standard mode and the fallback mode, using the remote console - give a green screen with "No Signal". I was pretty sure that this issue won't appear in fallback mode, but .. it does.

**Comment by shasheene** [comment]
[Rescuezilla v2.4.1](https://github.com/rescuezilla/rescuezilla/releases/latest) brought back Focal release, which may assist.

Yet unable to reproduce this on my local setup (all my machines use Nvidia, which works fine).

Rescuezilla v2.5 coming next month based on release with Ubuntu 22.10 (Kinetic), which may improve things too.

**Comment by engstk** [comment]
Focal seems to work normal with the devices I tested.
jammy only boots nomal with a RDNA2 gpu, RDNA and intel 10th igpu only work with nomodeset

**Comment by shasheene** [comment]
>  only boots nomal with a RDNA2 gpu, RDNA and intel 10th igpu only work with nomodeset

Super interesting. I wonder if Ubuntu 22.04 Jammy works? As mentioned I'll have an Ubuntu 22.10 "Kinetic" based build soon, and I can't reproduce this issue on my setup.

**Comment by engstk** [comment]
> > only boots nomal with a RDNA2 gpu, RDNA and intel 10th igpu only work with nomodeset
> 
> Super interesting. I wonder if Ubuntu 22.04 Jammy works? As mentioned I'll have an Ubuntu 22.10 "Kinetic" based build soon, and I can't reproduce this issue on my setup.

Yes, live media from Ubuntu 22.04, Linux Mint 21, other debian based distros, Manjaro 21.3.*, etc works as expected.

**Comment by Augusto7743** [comment]
That issue is related with Ubuntu 22.04 ?
I see users reporting same problems with some video cards and Nvidia GPUs 500 series.

**Comment by shasheene** [comment]
I think I've *finally* found the root cause why Rescuezilla has issues with Intel systems.**

Since v2.3 (released 2021-12-24), Rescuezilla ships with a [custom settings file `/etc/X11/xorg.conf.d/20-intel.conf` file](https://github.com/rescuezilla/rescuezilla/issues/281#issuecomment-1345457445) that was added by a user's suggest to prevent screen tearing on Intel graphics systems.

I've removed this file for the upcoming version of Rescuezilla coming very soon, which should fix the issues.
